### DISCUSSION TITLE
Adding Azure GOV to Power BI report that already has our main enrollment

### DISCUSSION THREAD

**Discussion by lundejd** [microsoft/finops-toolkit#602] [Q&A]
I have our main Azure enrollment working now using the setup and your Power BI example files. I'd now like to add Azure GOV as a FOCUS import since the document states "Designed to handle multiple accounts and clouds" 

I can do it using Cost Management API, but want to take advantage of FOCUS and stay consistent with measures etc.

Do I need to run the FinOps installer in our GOV tenant and have a new Storage URL, but where do I add that in Power BI?

**Comment by flanakin** [comment]
FOCUS support in Azure Gov is coming. We're waiting on Cost Management to ship their new exports release in Azure Gov. Once they do, you should be good. This is a good point, tho. I apologize for not internalizing that before because that means our Deploy to Azure Gov button for hubs does something that you can't use 😕 Same for Azure China.

I see 3 options:
1. Document the gap and link Azure Gov/China to 0.1.1 in the interim.
2. Build a converter to turn actual/amortized data to FOCUS in ADF.
3. Extend support for any type of export and do the FOCUS conversion in Power BI.

Option 1 is likely the most pragmatic for the next release (0.2.1).

Option 2 is a nice idea (and we did talk about that), but we won't be able to do FOCUS justice with the data that is available today. And since this is possibly throwaway work that should only be around for a month or two, I'm thinking it's not worth it. That is, unless you see some need for a FOCUS converter outside of this.

Option 3 is another one we've talked about and lines up with our long-term plans already. We've talked about this for 0.3 but it's not currently planned for that release. We can take a look at how much extra work that would be to determine if it's better to do that in 0.3 or 0.4. Ultimately, I suspect the timeline will be the same. It's just about whether we hold up the other features that are ready.

Thoughts?

**Reply by ro100e** [reply]
@flanakin Has there been any news on this? Just tried this with a gov customer and was not successful in creating a FOCUS export.

**Reply by flanakin** [reply]
Not yet. Check back in late June.

**Reply by flanakin** [reply]
@lundejd / @ro100e, would one of you mind creating a new issue to track the request for adding Azure Gov support? If I get enough upvotes, I might be able to use this to get it prioritized. It's a long shot, but worth trying.
